rules/game_process.tex

```diff
@@ -340,6 +416,53 @@ \subsection{Free Kick}
 Note that in the case of the Pushing free kick the sub-state is activated automatically through the ``Foul''.
 The team who is awarded the free kick (termed the attacking team) has \qty{\FreeKickTime}{\second} to complete the kick.
 
+\subsubsection{Visual gesture}
+\label{sec:free_kick_gesture}
+
+\textbf{In Champions Cup only}, the referee communicates which team is to take the free kick via a visual gesture.
+The GameController packet will tell the robots which free kick sub-state is active, but not who is to kick the ball.
+
+\begin{figure*}[t]
+  \centering
+  \begin{subfigure}{.33\textwidth}
+      \centering
+      \includegraphics[height=120px]{figs/visual-signal/kick-in.png}
+      \caption{Pointing to the left goal}
+  \end{subfigure}
+  \begin{subfigure}{.33\textwidth}
+      \centering
+      \includegraphics[height=120px]{figs/visual-signal/kick-in-flipped.png}
+      \caption{Pointing to the right goal}
+  \end{subfigure}
+  \caption{Visual signals for set plays.}
+\end{figure*}
+
+The gesture consists of raising one arm horizontally and sideways, parallel to the field line, while the other arm remains at rest.
+The raised arm is the one pointing towards the defending team's goal, i.e. the goal of the team that is \textit{not} taking the free kick.
+\todo[linecolor=gray,backgroundcolor=gray!25,bordercolor=gray]{
+  Remove the footnote after RC2025 has passed
+}
+\footnote{For reference, teams who participated in the 2023 Visual Referee Challenge
+  may notice that this the same as the "Kick-in" gesture from back then,
+  but with the opposite arm raised.
+}
+
+This gesture is made by the head referee and held for at least 5 seconds after the GameController signal is sent.
+The head referee is encouraged to seek the acknowledgement of the GameController operator and/or look at the field-facing monitor (if available) in order to determine the timing.
+
+The referee makes an effort to send the signal standing as close as possible to the T-junction joining the half-way line to the touchline opposite to the technical area.
```

Comment:
I think we can improve the wording to say that looking at the T-junction will allow the signal to be visible in the camera field (assuming it is not too close or positioned awkwardly at the extremity of the field). However, teams are not expected to scan the entire sideline.

Reply:
I'm pushing an additional sentence to this effect, but I'm open to discussion to phrase it better.